src/stories/SWGIcon.stories.mdx

```diff
@@ -0,0 +1,79 @@
+import { useState } from "react";
```

Comment:
Name of the file is SWGIcon.stories.mdx should be "SVG...

Reply:
done